File: python/025_raindrops/raindrops.py
def convert(number):
    s = ""    
    if not number % 3:
        s += "Pling"
    if not number % 5:
        s += "Plang"
    if not number % 7:
        s += "Plong"
    if not s:
        s += str(number)
    return s
        
        

File: typescript/033_raindrops/raindrops.ts
export function convert(n: number): string {
  let output: string = '';
  if(!(n % 3)) output += 'Pling';
  if(!(n % 5)) output += 'Plang';
  if(!(n % 7)) output += 'Plong';
  return output || n.toString();
}